static/src/js/routes/Project/Project.js

```diff
@@ -16,9 +16,10 @@ import ProjectCollectionsContainer
 import ProjectPanelsContainer from '../../containers/ProjectPanelsContainer/ProjectPanelsContainer'
 import OverrideTemporalModalContainer
   from '../../containers/OverrideTemporalModalContainer/OverrideTemporalModalContainer'
-import EdscMapContainer from '../../containers/MapContainer/MapContainer'
 import SavedProjectsContainer from '../../containers/SavedProjectsContainer/SavedProjectsContainer'
-import AuthRequiredContainer from '../../containers/AuthRequiredContainer/AuthRequiredContainer'
+import Spinner from '../../components/Spinner/Spinner'
```

Comment:
Map seems to be loading with some of the buttons missing on the projects page perhaps a CSS loading issue?